package.json

```diff
@@ -19,17 +19,17 @@
     "mocha": "mocha",
     "xo": "xo",
     "test": "npm run xo && npm run mocha",
-    "test:coveralls": "nyc npm run mocha"
+    "test:cover": "c8 npm run mocha"
```

Comment:
Since `npm run mocha` runs `mocha`, can this be simplified to `c8 mocha`?

Reply:
Yeah but I prefer reusing the existent scripts